interactive/templates/interactive/analysis_request_output.html

```diff
@@ -4,70 +4,125 @@
+          <h1 id="analysisHeading" class="mt-2 text-4xl font-bold text-slate-900 sm:text-5xl sm:tracking-tight lg:text-6xl">
+            {{ analysis.title }}
+          </h1>
+          <p class="text-sm font-bold text-oxford-600 tracking-wide uppercase">
+            Analysis Output
+          </p>
+        </div>
+        <p class="max-w-3xl mx-auto mt-5 text-xl leading-normal text-slate-700">
+          The aim of this OpenSAFELY Interactive report is to describe trends and variation in clinical codes to inform further analyses.
+          It is not intended to be a tool for "performance monitoring".
+        </p>
+      </div>
+
+      <div class="max-w-3xl mx-auto rounded-md bg-yellow-50 border border-yellow-300 p-4 pt-6 pb-7 mt-6">
+        <div class="flex">
+          <div class="flex-shrink-0">
+            {% include "icons/solid-exclamation.svg" with html_class="h-6 w-6 text-yellow-500" %}
+          </div>
+          <div class="ml-3 font-semibold text-yellow-800">
+            <p class="mb-3">Sharing outside of NHS England is not permitted in the current pilot phase without further approvals.</p>
+            <p>Please contact {% include "components/link.html" with href="mailto:team@opensafely.org" content="The OpenSAFELY Team" %} if you would like to seek approval for wider sharing.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+
+  <div class="container pt-8 py-12">
+    <section class="max-w-3xl mx-auto bg-white shadow overflow-hidden sm:rounded-lg text-slate-700">
+      <dl class="px-4 py-4 sm:px-6 text-sm grid grid-cols-1 gap-y-6">
+        <div>
+          <dt class="text-slate-600 font-semibold mb-1">Requested codelist</dt>
+          <dd>
+            {% include "components/link.html" with href="https://www.opencodelists.org/codelist/"|add:analysis.codelist content=analysis.codelist %}
+          </dd>
+        </div>
+        <div>
+          <dt class="text-slate-600 font-semibold mb-1">Time period</dt>
+          <dd>
+            {{ analysis.start_date }} to {{ analysis.end_date }}
+          </dd>
+        </div>
+      </dl>
+    </section>
+
+    <section class="px-4 sm:px-6 lg:px-8">
+      <div class="mt-8 flex flex-col max-w-3xl mx-auto">
+        <div class="-my-2 -mx-4 overflow-x-auto sm:-mx-6 lg:-mx-8">
+          <div class="inline-block min-w-full py-2 align-middle md:px-6 lg:px-8">
+            <div class="overflow-hidden shadow sm:rounded-lg">
+              <table class="min-w-full divide-y divide-slate-300">
+                <tbody class="divide-y divide-slate-200 bg-white">
+                  <tr class="divide-x divide-slate-200">
+                    <th scope="row" class="whitespace-nowrap px-4 py-3.5 text-left text-sm font-semibold text-slate-900">
+                      Unique events
+                    </th>
+                    <td class="whitespace-nowrap p-4 text-sm text-slate-600">
+                      {{ summary.0.count|intcomma }} This is some text
```

Comment:
> This is some text

This placeholder can be removed.

Table looks good with data in it!

Reply:
Oh yes, my really clever testing method there 😆